javascript/javascript3/week2/geolocation.js

```diff
@@ -0,0 +1,31 @@
+const setTimeoutPromise = (timeout) =>{
+     let promise = new Promise((resolve, reject) => {
+       let seconds = timeout * 1000;
+       setTimeout(() => {
+         resolve("Called after " + timeout + " seconds");
+       }, seconds);
+     })
+       .then((result) => console.log(result))
+       .catch((error) => console.log("error"));
+
+}
+//setTimeoutPromise(3); 
+
+
+
+
+
+  async function getCurrentLocation(postion) {
+    // const latitude = postion.coords.latitude;
+    // const longitude = postion.coords.longitude;
+    try {
```

Comment:
Here you are using both try/catch and a .then() function. This gets a bit confusing.